scripts/build-bugzoo.py

```diff
@@ -11,10 +11,16 @@
 DIR_HERE = os.path.dirname(__file__)
 DIR_ROBUST = os.path.abspath(os.path.join(DIR_HERE, '..'))
 
+IGNORE_FILENAME='ROBUST_IGNORE'
 
-def find_bug_descriptions(d):
+def find_bug_descriptions(d, ignore_marker=IGNORE_FILENAME):
     buff = []
-    for root, _, files in os.walk(d):
+    for root, dirnames, files in os.walk(d):
+        if ignore_marker in files:
+            logger.info("Ignoring {} (and children) because of ROBUST_IGNORE".format(root))
```

Comment:
If you want to be really clever, you could swap `ROBUST_IGNORE` with a template argument (i.e., `{}` and `IGNORE_FILENAME`).

Reply:
Initially `ignore_marker` was hard-coded to always be `ROBUST_IGNORE`, then I made it a defaulted arg but forgot to update the log message.

Should be fixed now.